.test/config-sra/config.yaml

```
@@ -112,4 +112,9 @@ tables:
     genotype: true
     coverage: true
     event_prob: true
-  generate_excel: true
\ No newline at end of file
+  generate_excel: true
+  
+custom_alignment_properties:
+  activate: false
+  column: "panel"
+  tsv: ""
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)